### ISSUE TITLE
editor: names with other language characters can't be exported

### ISSUE THREAD

**Issue by tehKaiN** [drom/wavedrom#137]
Try to export to SVG signal named as following: "zażółć gęślą jaźń" or any of this sentence.

**Comment by drom** [comment]
aahh, yes

```
Uncaught InvalidCharacterError: Failed to execute 'btoa' on 'Window':
The string to be encoded contains characters outside of the Latin1 range.
saveSVG @ editor.js:195
onclick @ editor.html:30
```

**Comment by drom** [comment]
Fixed in v1.4.2
https://github.com/wavedrom/wavedrom.github.io/releases/tag/v1.4.2
